styles/style.css

```diff
@@ -0,0 +1,80 @@
+h1 {
+  font-family: 'Noto Serif JP', sans-serif;
+}
+
+ul {
+  list-style-type: none;
+}
+
+nav ul li:hover {
+  text-decoration: underline;
+  color: white;
+}
+nav {
+  background-color: silver;
+}
+
+section {
+  border-bottom: 1px solid grey;
+}
+
+.navigation {
+    border: 1px solid grey;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+}
+
+nav ul li:nth-child(2) {
+  grid-column: 3 / 4;
+}
+
+.why {
```

Comment:
great use of flex.. maybe put the entire section in a 2 row grid (h2 / other)...  and use flex on the 2nd row (image/text section)